topics/dev/tutorials/tool-annotation/tutorial.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: tutorial_hands_on
+title: Annotation of a tool with EDAM ontology terms using bio.tools
```

Comment:
Alternate title suggestion: `Linking Galaxy tools to metadata in the bio.tools registry`

Reply:
I feel that we cover more than just `Linking`  in the tutorial. We cover also the aspect of creating bio.tools and improving the annotation on bio.tools

Reply:
What about: `Adding and updating best practice metadata for Galaxy tools using the bio.tools registry`?